graphics/mesa/file_contexts

```diff
@@ -22,9 +22,12 @@
 /(vendor|system/vendor)/lib(64)?/libskuwa\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/hw/gralloc\.broxton\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/hw/gralloc\.intel\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/hw/gralloc\.minigbm\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/libgrallocclient\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/libglapi\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/dri/i965_dri\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/dri/virtio_gpu_dri\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/dri/gallium_dri\.so u:object_r:same_process_hal_file:s0
```

Comment:
I think we haven't used gallium_dri.so

Reply:
When we enable virtio, the gallium_dri.so is built out. Maybe gallium_dri.so is one dependent library for virtio?

Reply:
for virtio gpu implementation, we need virtio_gpu_dri.so. Gallium_dri.so is sw opengl implementation. In Celadon, currently we use swiftshader instead of gallium_dri.so.

Reply:
my mistake, gallium_dri.so is not for software opengl implementation, it is necessary to work with virtio_gpu_dri.so